memory/src/main/scala/filodb.memory/format/vectors/Histogram.scala

```diff
@@ -160,6 +262,9 @@ final case class LongHistogram(buckets: HistogramBuckets, values: Array[Long]) e
              s"Expected: ${buckets}, Found: ${other.buckets}"
       )
     }
+    // TODO if otel histogram, the need to add values in a different way
+    // see if we can refactor since MutableHistogram also has this logic
+    assert(other.buckets == buckets)
```

Comment:
nitpick: wondering if this check is redundant, since the previous if check does the same thing ?

Reply:
ah yes - this is from a merge conflict that I needed to resolve. I'll remove in my next PR